lua/orgmode/config/defaults.lua

```diff
@@ -133,6 +136,10 @@ return {
       org_set_effort = '<leader>oxe',
       org_show_help = 'g?',
     },
+    edit_src = {
```

Comment:
I think there is no need to nest this into `edit_src` object, we could have it on top level.

Reply:
It gets a bit confusing as _Vim_ gets confused when using `%` to hop between matching `{`/`}` pairs, but this is just under the `mappings` section. Is that where it should be?

Reply:
No, I actually thought to have it on the same level as other org mappings, but then I figured out this is a separate buffer so it needs to be split. Everything is ok.